File: websocket.js
const socket = new WebSocket('wss://cekirdektenyetisenler.kartaca.com/ws');

socket.addEventListener('message', event => {
  // process the incoming message
  const data = JSON.parse(event.data);
  console.log(data);
});

socket.addEventListener('open', event => {
  console.log('WebSocket connection established');
});

socket.addEventListener('close', event => {
  console.log('WebSocket connection closed');
});

socket.addEventListener('error', event => {
  console.error('WebSocket error:', event);
});

File: websocket.py
import websocket
import json


def on_message(ws, message):
    # process the incoming message
    data = json.loads(message)
    print(data)


# create a WebSocket connection to the URL
ws = websocket.WebSocketApp("wss://cekirdektenyetisenler.kartaca.com/ws")

# set the message handler
ws.on_message = on_message

# start the WebSocket connection
ws.run_forever()
